lib/problem2.rb

```diff
@@ -0,0 +1,17 @@
+def problem2(values)
+  raise ArgumentError, 'not enough elements for operation' if values.count < 2
+
+  first = nil
+  second = nil
+
+  values.each do |value|
+    if value > first.to_i then
+      second = first
+      first = value
+    elsif value > second.to_i then
+      second = value
+    end
+  end
+
+  return first, second
```

Comment:
As [rubocop](https://github.com/rubocop-hq/rubocop) will put it: `redundant return statement.`